subscription-engine/kafka-ws/test/unit/WebsocketTarball.test.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Unit test for websocketTarballs Module
+ */
+
+/**
+  * mocks
+  */
+jest.mock('../../app/Kafka/KafkaTopicsConsumerCallbacksMgmt');
+jest.mock('../../app/Redis/RedisExpireMgmt');
+
+const mockMicroServiceSdk = {
+  ConfigManager: {
+    getConfig: jest.fn(() => jest.fn()),
+    transformObjectKeys: jest.fn((obj) => obj),
+  },
+  Kafka: {
+    Consumer: jest.fn(),
+  },
+  ServiceStateManager: jest.fn(() => ({
+    registerService: jest.fn(),
+    signalReady: jest.fn(),
+    signalNotReady: jest.fn(),
+    addHealthChecker: jest.fn((service, callback) => callback(jest.fn(), jest.fn())),
+  })),
+  Logger: jest.fn(() => ({
+    debug: jest.fn(),
+    error: jest.fn(),
+    info: jest.fn(),
+    warn: jest.fn(),
+  })),
+};
+
+jest.mock('@dojot/microservice-sdk', () => mockMicroServiceSdk);
+
+const webSocketTarball = require('../../app/WebsocketTarball');
+
+describe('Test websocket tarball', () => {
+  beforeEach((done) => {
+    jest.resetAllMocks();
+    done();
+  });
+
+  beforeAll((done) => {
+    done();
+  });
+
+  it('should init the websocket sucessfully', () => {
+    webSocketTarball.init();
+  });
+
+  it('should test onClose callback', () => {
+    webSocketTarball.onClose();
+  });
```

Comment:
These are some strange tests, they do nothing. Are you obliged by codecov to do this because of the total coverage? Can't you do tests for another class that is missing tests if it's the case? If you are doing because of codecov/patch, you can leave it; codecov/patch is not really obligatory.